frontend/src/components/authForm/InputControl.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useField } from 'react-final-form';
+import {
+	FormControl,
+	FormLabel,
+	FormErrorMessage,
+	FormHelperText,
+	Input,
+} from '@chakra-ui/react';
+
+const InputControl = ({ name, label, type, helper }) => {
+	const { input, meta } = useField(name);
+	return (
+		<FormControl name={name} mb={4} isInvalid={meta.error && meta.touched}>
+			<FormLabel htmlFor={name}>{label}</FormLabel>
+			<Input {...input} name={name} type={type} />
```

Comment:
`{...input}`  here allows me to access HTML input props into the component so that I can use it with React Final Form. Without it, validation doesn't work properly.